### ISSUE TITLE
Are there any finetuned open-source models that the paper mentioned?🤗

### ISSUE THREAD

**Issue by oushu1zhangxiangxuan1** [BeachWang/DAIL-SQL#4]

**Comment by BeachWang** [comment]
Hi,

Thank you for your interest in our work. In this work, we have conducted insightful experiments on fine-tuned open-source models. However, certain challenges, such as the loss of in-context learning capacity, remain open questions. We are actively pursuing solutions in this area and will release our model once we have successfully addressed this issue.

**Comment by vincent317** [comment]
Hello,

Thank you for the great work!

Quick question: When you mean the loss of in-context learning capacity for finetuned models, are these models finetuned on few-shot data as well or zero-shot data only?
